### PR TITLE
UCT/UCS: Fix RoCE LAG detection

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucs/sys/sock.c [hunk 1]
```diff
@@ -82,6 +83,22 @@ int ucs_netif_is_active(const char *if_name)
     return ucs_netif_flags_is_active(ifr.ifr_flags);
 }
 
+unsigned ucs_netif_bond_ad_num_ports(const char *bond_name)
+{
+    ucs_status_t status;
+    long ad_num_ports;
+
+    status = ucs_read_file_number(&ad_num_ports, 1,
```
Comment (dmitrygx) [inline src/ucs/sys/sock.c]: http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/label=hpc-test-node-legacy,worker=0/14857//artifact/cov_build_devel_14857/output/errors/1/1sock.c.html#error

what about the following fix?
```
    ...
    ucs_assert_always(ad_num_ports <= UINT_MAX);
    return (unsigned)ad_num_ports;
}
```

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 7 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 16-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/2ea7a0309110579d725b1a76763d9d5d33db404a/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/a23a7170547899c81111b556201cf31ad6e5598b/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/1260ad01523b26161f7b2893a4b6206f54cb9bf8/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/bfea0cb666d4caf4d94ef7c09085adc8fb807389/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/403be59f41819628e807538a45635799d38774c3/hpc-test-node-legacy_W3) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/4594d8c58f01dddf1fc203223540d8d518bab237/hpc-test-node-new_W0) | :x: FAILURE
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/eda3aee89a6c65e9ea5d326ee1805f18095fb896/hpc-test-node-new_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/069c610ba49869c4f1315e8f217cda5072a23110/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/63bb571423ec1a8bcb632a1d31b38a51594b71eb/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/cbd6f050b99c3dc035abf1523911ef0eea21e111/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/24e8973e05f63081e2f58ad6ade4ea282b9432a4/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/63dd690ddb0e3f681a10b7038bcf6185e267c191/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/ecc7e38be5a8b15ba1130b4a48af2f6d30bb176d/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/97dc1d3a767f081ae8c0a704a7de3b7445f21984/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/342c3fe33eee24c2c789d0d4e4e1d3ff52320ef4/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/3ad6c0f451528d0a00273571cdd9da073456a2ec/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/007d30ae72e5f454a1f4f55ee2fe214ebf58db7d/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/27d058de084686c87ba6c3cb3fb1ce4a8688ce3d/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/f3e790b7db0419d6c7681ced1ed20e933ba3b1c6/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/00e642fdc771c9443b1541df8797439712fefc77/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/0822153d77a9e01c891ffe43999e4aecf5ffabb1/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/e842c45a65cd5f0943c54aff1a0eaf640fe25201/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/aeb09c3948e0148b5840caf8fdd3f1566f5bd5b0/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/4e094e5f0b326f87c95b0c643c04c9c90eac29fa/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/85e79348d38a2763f71183470deccbe5/raw/f1dd4ebd97906a90a5418097266f1f90570ab2f6/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/ucs/sys/sock.c [hunk 2]
```diff
@@ -90,13 +90,14 @@ unsigned ucs_netif_bond_ad_num_ports(const char *bond_name)
 
     status = ucs_read_file_number(&ad_num_ports, 1,
                                   UCS_NETIF_BOND_AD_NUM_PORTS_FMT, bond_name);
-    if (status != UCS_OK) {
-        ucs_diag("failed to read " UCS_NETIF_BOND_AD_NUM_PORTS_FMT ": %m, "
+    if ((status != UCS_OK) || ((ad_num_ports <= 0) ||
+        (ad_num_ports > UINT_MAX))) {
```
Comment (dmitrygx) [inline src/ucs/sys/sock.c]: No need for additional `()` around `ad_num_ports` check
Or is this for readability?

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 16-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/0e8f733683464b9483825c530af36006f2a27529/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/dca1dfe43cc983eb71174cda1ab263051439511a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/19b41058115c5f1c389330eee8f69bb1469636ae/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/b6311a874856fd0fb8adfb1fb678e66e4c0781bb/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/c6127cc6eb4509c712346dcda8c35f007c84c744/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/12ff3ec93fb48c144b2ca3c2dc511dfab3be0163/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/b5cb4e5e40ef6e0f4f78a6d2631a04eb817adff6/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/cb2d5faf36479286c959acdadc8bb569706ba41a/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/90808d624a9adc49265fee8a0d95a4711c0cdbd1/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/bd05e52e9402df166c6464543e9a0fe818de483d/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/e94362a6d29371511ef9ea330efdec044c629b75/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/f32cd5d6be06363a6e4fbd65f6c2005f949df03f/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/e14f423fb1d997e8f03fcc7c077ae3184f2dcdc1/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/1cee9ce7cc4a56c76d86e1f58969825bf91d559a/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/b22ad3787405c6eff0f004e2a4875ebc97b2f579/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/bbdc4c0845ee124d21efe37523452c2b218e4657/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/ae09a3047460b0632ebbd77aa667bcb362f56f07/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/acdf14bc99c94474b386c0bc614c8dc7a268a618/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/d8aecc175a2480bd958b41c3af8e65e76fbcd5d2/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/0f28b1c016f1010911def4d221f1216103ed6fbb/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/9ba0a1545a01718e887caf36efa61096b265ca4c/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/c3353b955d43ed6f19c1db8bd535f5a1efc0f030/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/c03ac884933ada2f083cb09cc1c93b5ba20abe18/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/eb45f4fc0304f33ba9dc1d78a125eb839186fcfc/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/2ca8c00e690c7727dab72b56649b3d80/raw/fce4d2f845c418ee5cd8ebdc8c89d5381135bd62/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by dmitrygx** [comment]
`r-vmb-ubuntu17-u10-x86-64-jb` was offline
bot:pipe:retest

File: src/ucs/sys/sock.c (hunk 2 above)
Comment (yosefe) [reply src/ucs/sys/sock.c]: @dmitrygx it was a copy/paste issue, fixed. 10x!

Comment (dmitrygx) [reply src/ucs/sys/sock.c]: > @dmitrygx it was a copy/paste issue, fixed. 10x!

;)

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 17-Mar-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/4eec478f9d8fbeda07fc354cf2751d3dd4113891/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/10e2795c520897c30fe3a7904b00a7f8d414e704/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/49f1e46f8df3ce9ea9955062d751481f7b4d14fc/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/6c460f3e918cf8f4787741814eced307d3013fac/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/a95175e35ec56fa451853efb01f4572d1242c65c/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/cfe0313ec5d8075e5c6796d1b80767358fd380d9/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/742ae6b49f0da4e753bc07fbcaaa487b441108a9/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/5775d61047dc761004bfe93fcf2ca6a6cfde6487/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/7f9f285e5a851b6a080e7c13b1c2717870141193/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/03fa185425f0e51906aceec0a10d3561ea3324bc/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/2b1c55371684c01f3cf1254ea7fb59d7e47e5391/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/476d2a9a256a37c451227a13c8c522b46b79608a/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/f72b010f49becd243602701204017cd56657b64a/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/eb3645bf255d73ba9378adc2b550b8c07403baf9/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/f6098045848f65b935d7605b990f425d11d8fab0/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/066ddda7f409e2526065bc4ac0b61b69ae813824/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/8e128e07236759e6d0c2f6c76d4705a1bff063ef/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/3790ffaf51408a212f3467498f6780e81057abe4/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/953d5983c148ef3d6a9f5a36eb727a05be3c8ce1/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/ce645c191807a274cadecc9dc423be2c7abf292e/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/c42050b4077cf468c2df59b03953ebd79daf8fac/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/59399480ba748b30eecd3d842f797af7a716b7e0/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/533cdc6688dd801b9823fe69a85c27193836cf01/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/c512bbc48eeea2ee500394e9182747b76cd30ebe/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/337d039a3150b27e1e58c9d926ed45f8/raw/637a28a53045d4bb06c42a02f2d3306434c9bf8e/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**